command/volume_delete.go

```diff
@@ -0,0 +1,99 @@
+package command
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/nomad/api/contexts"
+	"github.com/posener/complete"
+)
+
+type VolumeDeleteCommand struct {
+	Meta
+}
+
+func (c *VolumeDeleteCommand) Help() string {
+	helpText := `
+Usage: nomad volume delete [options] <vol id>
+
+  Delete a volume from an external storage provider. The volume must
+  not be currently claimed by an allocation.
```

Comment:
I'd suggest documenting what happens if they claimed? does it get detach from alloc? deletion request fails?

Reply:
It fails with an error. That's in the docs... will copy that section here too.